processor_function/shared/cnab_builder.py

```diff
@@ -86,24 +105,26 @@ def _setup_aci_deployment(self) -> ContainerGroup:
         container = Container(name=self._container_group_name,
                               image=container_image_name,
                               resources=container_resource_requirements,
-                              command=self._build_porter_cmd_line().split(),
+                              command=self._build_porter_cmd_line(),
                               environment_variables=self._build_cnab_env_variables())
 
         group = ContainerGroup(location=self._location,
                                containers=[container],
                                image_registry_credentials=image_registry_credentials,
                                os_type=OperatingSystemTypes.linux,
-                               network_profile=self._get_network_profile(),
+                               # network_profile=self._get_network_profile(),
```

Comment:
Why is this commented out? Is it coming back at some point? The story does not tell.

Reply:
It's maybe, probably, coming back, but not before MSI support in VNET ACI, removed the whole thing for now.